### DISCUSSION TITLE
How to make an exe with this package?

### DISCUSSION THREAD

**Discussion by Anirudh-12** [Akascape/CTkMessagebox#16] [Q&A]

**Comment by Akascape** [comment]
@Anirudh-12 
Pack it just like the *customtkinter* folder.

**Steps:**
- Copy the **ctkmessagebox** folder in the directory where your program is present. (Optional step, sometimes needed)

- Add that folder within the `additional folder` option of pyinstaller (auto-py-to-exe).

- Compile again to exe.

Detailed guide is given here: https://github.com/TomSchimansky/CustomTkinter/discussions/939
